yt/frontends/ramses/data_structures.py

```diff
@@ -312,12 +302,7 @@ def amr_offset(self):
 
     @property
     def max_level(self):
-        self._read_amr_header()
-        return self._max_level
-
-    @max_level.setter
-    def max_level(self, value):
-        self._max_level = value
+        return self.ds.max_level
```

Comment:
See https://github.com/cphyc/yt/blob/6ed3f95b20c10a33d9c93580a8d89c22e388be36/yt/frontends/ramses/data_structures.py#L1104 for the definition.